src/cbmc/cbmc_parse_options.h

```diff
@@ -63,7 +63,7 @@ class optionst;
   "(java-cp-include-files):" \
   "(localize-faults)(localize-faults-method):" \
   "(lazy-methods)" \
-  "(test-invariant-failure)" \
+  "(test-invariant-failure):" \
```

Comment:
I admit I have no idea how this works: all code handling "test-invariant-failure" has been removed. So where is it being processed?